products/sle/main.pm

```diff
@@ -572,6 +573,9 @@ sub load_reboot_tests() {
         }
         if (get_var('ENCRYPT')) {
             loadtest "installation/boot_encrypt";
+            if (check_var('BACKEND', 'svirt')) {
+                loadtest "installation/reconnect_s390";
+            }
```

Comment:
you should also do a check_var('ARCH', 's390x') to prevent it of being executed on other svirt jobs